website/docs/getting-started/setup/instance-configuration/http-proxy.md

```diff
@@ -0,0 +1,34 @@
+---
+description: This page provides steps to configure HTTP/HTTPS proxy on your self-hosted instance.
+---
+
+# Configure Proxy
```

Comment:
Let's keep to `Configure HTTP/HTTPS Proxy`, and not include the word `Server` please. Otherwise it sounds like we're about to talk about configuring the server itself. 🤔